flakey-test-tracking/src/github.js

```diff
@@ -0,0 +1,367 @@
+const { Octokit } = require('@octokit/rest')
+const { App } = require('@octokit/app')
+const { retry } = require('@octokit/plugin-retry')
+const Client = Octokit.plugin(retry)
+const stripAnsi = require('strip-ansi')
+const config = require('./config')
+const utils = require('./utils')
+
+// This is the FlakeTracker GitHub App ID. Can be found at github.com -> celo-org -> app settings
+const flakeTrackerID = 71131
```

Comment:
It's not a problem to have the Id here, right?
Maybe we could add it in the config file

Reply:
No not a problem at all, it's publicly accessible via the API. It would be better to have it in the config file though.